server/server.js

```diff
@@ -25,11 +28,52 @@ app.use(cors());
 app.get('/api/photos', (req, res) => {
   let id = req.query.product_id;
   console.log('prod id --->', req.query.product_id);
-  db.getAllPhotos(1, (err, result) => {
+  db.getAllPhotos(id, (err, result) => {
     res.json(result);
   });
 });
 
+app.post('/api/photos', (req, res) => {
+  db.savePhotos(req.body, (err, result) => {
+    if(err) {
+      console.log('error with saving to db', err)
```

Comment:
this is where you could get a little more detailed info from your errors I mentioned earlier if you pass it up the callbacks. Not essential but may be helpful down the road.